client-react/src/pages/app/app-settings/ErrorPages/ErrorPageFileUpload.tsx

```diff
@@ -46,7 +46,7 @@ const ErrorPageFileUploader: React.FC<ErrorPageFileUploaderProps> = (props: Erro
       const file = event.target.files[0];
       setFileName(file.name);
 
-      if (file.type !== t('customErrorPages_fileType') || file.size > 10000) {
+      if (file.type.includes(t('customErrorPages_fileType')) || file.size > 10000) {
```

Comment:
1. This should be an error if not `text/html`.
2. `customErrorPages_fileType` should be a constant, not a resource string.
3. Consider using `startsWith` instead of `contains`.

```suggestion
      if (!file.type.startsWith(t('customErrorPages_fileType')) || file.size > 10000) {
```

Reply:
1. I didn't thing the changes had synced in at that time
2. Okay I could change that
3. okay